packages/frontapp/src/app/components/BookInfo/index.tsx

```diff
@@ -3,10 +3,16 @@ import bookImage from '../../../assets/MOC-data/BookImage.png';
 import React, { FC, useCallback, useEffect, useState } from 'react';
 import { colors, dimensions } from '@mimir/ui-kit';
 import { ReactComponent as Claim } from '../../../assets/claim.svg';
+import { ReactComponent as Edit } from '../../../assets/Edit.svg';
```

Comment:
You need to replace the icon with the one in Figma. I already talked to the designer about some export issues, but I couldn't reproduce them. Looking at the svg code itself, it also looks normal

Reply:
I've changed it